src/data/d2/crafting-resonant-elements.ts

```diff
@@ -1,11 +1,7 @@
-export const resonantElementTagsByObjectiveHash: Record<number, string> = {
-  4061659983: 'resonant', // Resonant Elements
-} as const;
+export const resonantElementTagsByObjectiveHash: Record<number, string> = {} as const;
```

Comment:
We'll want to fix this up maybe?

Reply:
resonant elements have been deprecated. We should just no longer pull this file in.

Reply:
Resonant elements are 100% gone anyway but we need to make some changes to fix up deepsight, yes.